src/lua/api-gateway/validation/factory.lua

```diff
@@ -101,9 +102,9 @@ local function _generateHmacSignature()
     return hmacSignatureValidator:generateSignature()
 end
 
-local function _validateOAuthToken()
+local function _validateOAuthToken(obj)
     local oauthTokenValidator = OAuthTokenValidator:new()
-    return oauthTokenValidator:validateRequest()
+    BaseValidator:exitFn(oauthTokenValidator:validateRequest(obj))
```

Comment:
the other `_validate*` methods don't use `exitFn` here. If we want to change this then it would be easier to maintain this behavior if we  change the others as well. What was the reason for this change here ? Was it behaving incorrectly ?